tests/test_inputfile.py

```diff
@@ -17,16 +17,16 @@
 # You should have received a copy of the GNU Lesser Public License
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 import logging
-import os
 import subprocess
 import sys
 from io import BytesIO
+from pathlib import Path
 
 from telegram import InputFile
 
 
 class TestInputFile:
-    png = os.path.join('tests', 'data', 'game.png')
+    png: Path = Path('tests/data/game.png')
```

Comment:
e.g. what is the annotation good for here? should be clear that `png` is of type `Path` …

Reply:
removed annotation